src/mine.rs

```diff
@@ -81,13 +81,18 @@ impl Miner {
         // Dispatch job to each thread
         let progress_bar = Arc::new(spinner::new_progress_bar());
         progress_bar.set_message("Mining...");
+        let rt = tokio::runtime::Handle::current();
+        // inside the rayon threadpool the tokio runtime is not available.
+        // we access the runtime where it is available (outside of the pool) and spawn tasks directly on that runtime.
         let handles: Vec<_> = (0..threads)
+            .into_par_iter()
             .map(|i| {
-                std::thread::spawn({
-                    let proof = proof.clone();
-                    let progress_bar = progress_bar.clone();
-                    let mut memory = equix::SolverMemory::new();
-                    move || {
+                rt.spawn({
```

Comment:
Same as the above review.

Reply:
[`d894bb4` (#103)](https://github.com/regolith-labs/ore-cli/pull/103/commits/d894bb433a3648ff123c8fe32b3057523a387d0f)